src/System.CommandLine/Parsing/StringExtensions.cs

```diff
@@ -204,6 +204,14 @@ bool CanBeUnbundled(string arg, out IReadOnlyList<string>? replacement)
                     return false;
                 }
 
+                // don't unbundle if this is a known option
+                if (knownTokens.Values
```

Comment:
Does `knownTokens.ContainsKey(arg)` also work? It will be more performant.

Reply:
Sure, just made that change.